content/software/ide-v2/tutorials/ide-v2-serial-monitor/ide-v2-serial-monitor.md

```diff
@@ -36,12 +38,16 @@ Now, let's take a look at the IDE 2.0. Notice how the Serial Monitor is located
 
 The Serial Monitor settings are also located here, such as adjusting the baud rate and sending messages. 
 
-### Advantages with New Integration
+#### Advantages
 
 A major advantage with having the Serial Monitor integrated with the editor is the possibility to have multiple monitors open simultaneously. In the old editor, when changing the port/board, it changed across all windows. In addition, we were limited to **one Serial Monitor window**, an obstacle that the IDE 2.0 removes. 
 
 You will find an example with more information on how to use this feature, further down this tutorial under the **Using multiple Serial Monitors simultaneously** section.
 
+### Different Send Trigger
+
+In the Arduino IDE 1.x Serial Monitor, data can be sent to the board by entering it in the field at the top of the window and then clicking the **Send** button or pressing the <kbd>**Enter**</kbd> key. In the Arduino IDE 2.0 Serial Monitor you must instead use the <kbd>**Ctrl**</kbd>+<kbd>**Enter**</kbd> keyboard shortcut (<kbd>**Command**</kbd>+<kbd>**Enter**</kbd> for macOS users).

```

Comment:
@per1234 I'm not sure how the <kbd> tag will look like in the published docs, and this is not something we use normally. However I think we can merge this PR anyway, as I think the changes look good.

Reply:
@jhansson-ard I followed a precedent from existing content:

https://github.com/arduino/docs-content/blob/6f57afe6ca4119417af5bf369b21023c61e0b136/content/hardware/03.nano/carriers/nano-motor-carrier/tutorials/nano-simulink-wifi-led/nanoSimulinkWiFiLED.md#103

https://github.com/arduino/docs-content/blob/6f57afe6ca4119417af5bf369b21023c61e0b136/content/hardware/03.nano/carriers/nano-motor-carrier/tutorials/nano-simulink-wifi-led/nanoSimulinkWiFiLED.md#149

You can see how the markup is rendered in that published tutorial here:

https://docs.arduino.cc/tutorials/nano-motor-carrier/nano-simulink-wifi-led

Not so interesting to be sure. I can remove the markup if you like.

Reply:
Well then! Let's leave them in, I think it looks great ⭐